triangle.c

```diff
@@ -244,16 +301,7 @@ int main(int argv, char **argc)
                  buffer);
 
     // Write all pixels to file
-    FILE *output = fopen("triangle.raw", "wb");
-    if (output)
-    {
-        fwrite(buffer, 1, desiredWidth * desiredHeight * 3, output);
-        fclose(output);
-    }
-    else
-    {
-        fprintf(stderr, "Failed to open file triangle.raw for writing!\n");
-    }
+    int success = stbi_write_png("triangle.png", desiredWidth, desiredHeight, 3, buffer, desiredWidth * 3);
```

Comment:
A small nit, you are not using the `success` result.

Could you add something like the following after?

```
if (success != 1) {
    fprintf(stderr, "Failed to open file triangle.png for writing!\n");
}
```